docs/apis/_files/db-access-php.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Moodle Pty Ltd.
+ *
+ * Moodle is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Moodle is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import {
+    fillDefaultProps,
+    getDescription,
+    getExample,
+    PluginFileSummary,
+} from '../../_utils';
+import type { Props } from '../../_utils';
+import DefaultDescription from './db-access-php.mdx';
+
+const defaultExample = `$capabilities = [
+    // Ability to use the plugin.
+    'plugintype/pluginname:useplugininstance' => [
+        'riskbitmask' => RISK_XSS,
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => [
+            'manager' => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+        ],
+    ],
+];
+`;
+
+export default (initialProps: Props): PluginFileSummary => {
+    const props = fillDefaultProps({
+        filepath: '/db/access.php',
+        summary: 'Plugin capabilities',
```

Comment:
```suggestion
        summary: 'Plugin capabilities',
        refreshedDuringUpgrade: true,
```

Reply:
Shouldn't the Upgradable badge be displayed for db/access.php too? Capabilities are also updated when the upgrade is performed...

Reply:
Nope - they aren't refreshed!! New ones are _added_ but existing ones are not modified.